cartographer/mapping_2d/pose_graph/spa_cost_function_2d.h

```diff
@@ -30,25 +30,23 @@
 #include "ceres/jet.h"
 
 namespace cartographer {
-namespace mapping_2d {
+namespace mapping {
 namespace pose_graph {
 
-class SpaCostFunction {
+class SpaCostFunction2D {
  public:
-  using Constraint = mapping::PoseGraph::Constraint;
```

Comment:
unrelated change?

Reply:
After removing 'mapping::' from this typedef, there seems to be no reason to keep it at all. Inlined it.